### PR TITLE
DEC-1370 feat: port queryapi feed -> near-discovery-components

### PR DESCRIPTION
**Pull request by roshaans**
near.org feed powered with QueryAPI

### CODE REVIEW

**Review by shelegdmitriy — CHANGES_REQUESTED** [review]

File: src/Feed.jsx [hunk 1]
```diff
@@ -1,29 +1,122 @@
...
+
+let filterUsersRaw = Social.get(
+  `${moderatorAccount}/moderate/users`,
+  "optimistic",
+  {
+    subscribe: true,
+  }
+);
+
+if (filterUsers === null) {
```
Comment (mpeterdev) [inline src/Feed.jsx:22]: this should be `filterUsersRaw`, it was an existing bug

File: src/Posts.jsx [hunk 2]
```diff
@@ -1,32 +1,235 @@
...
 if (previousSelectedTab && previousSelectedTab !== state.selectedTab) {
   State.update({
     selectedTab: previousSelectedTab,
   });
 }
 
-let accounts = undefined;
-
-if (state.selectedTab === "following" && context.accountId) {
+// TODO port to QueryAPI
```
Comment (mpeterdev) [inline src/Posts.jsx:23]: this is intentionally left for later, right?

File: src/Posts.jsx [hunk 3]
```diff
@@ -1,32 +1,235 @@
...
+  }
+);
+
+const selfFlaggedPosts = context.accountId
+  ? Social.index("flag", "main", {
+      accountId: context.accountId,
+    })
+  : [];
+
+if (filterUsers === null) {
```
Comment (mpeterdev) [inline src/Posts.jsx:61]: should also be `filterUsersRaw`

**Review by mpeterdev — CHANGES_REQUESTED** [review]
code changes LGTM, only change requested is fixing the existing bug I pointed out while you're in here

still need to do more functional testing, currently seeing `VM is dead errors` when running with bos-loader

File: src/Posts.jsx (hunk 2 above)
Comment (roshaans) [reply src/Posts.jsx:23]: Yes. We fetch the followers using the `Social` API as we do not yet index follower with QueryAPI yet.

**Review by mpeterdev — APPROVED** [review]

**Pull request merged**